packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -395,11 +395,11 @@ contract Multisig is PluginUUPSUpgradeable, ProposalUpgradeable, Addresslist {
 
     /// @notice Internal function to update the plugin settings.
     function _updateMultisigSettings(MultisigSettings calldata _multisigSettings) internal {
-        uint256 addresslistLength_ = addresslistLength();
+        uint16 addresslistLength16 = addresslistLength().toUint16();
```

Comment:
same as above.

Reply:
After a discussion with @novaknole (https://discord.com/channels/672466989217873929/1063135237699289150/1063503000632569936), we agreed to convert both, `minApprovals` and `addresslistLength`, to `uint32`.